src/Configuration.ts

```diff
@@ -0,0 +1,491 @@
+          description: "Minimize test cases at the end of the search",
+          group: "Postprocess options:",
+          hidden: false,
+          type: "boolean",
+        })
+    );
+  }
+
+  configureSamplingOptions<T>(yargs: Yargs.Argv<T>) {
+    return (
+      yargs
+        // random number generator settings
+        .option("seed", {
+          alias: ["s"],
+          default: null,
+          description: "Seed to be used by the pseudo random number generator.",
+          group: "Sampling options:",
+          hidden: false,
+          type: "string",
+        })
+        // sampling settings
+        .option("max-depth", {
+          alias: [],
+          default: 5,
+          description: "Max depth of an individual's gene tree.",
+          group: "Sampling options:",
+          hidden: false,
+          type: "number",
+        })
+        .option("max-action-statements", {
+          alias: [],
+          default: 5,
+          description:
+            "Max number of top level action statements in an individual's gene tree.",
+          group: "Sampling options:",
+          hidden: false,
+          type: "number",
+        })
+        .option("constant-pool", {
+          alias: [],
+          default: false,
+          description: "Enable constant pool.",
+          group: "Sampling options:",
+          hidden: false,
+          type: "boolean",
+        })
+        // mutation settings
+        .option("explore-illegal-values", {
+          alias: [],
+          default: false,
+          description:
+            "Allow primitives to become values outside of the specified bounds.",
+          group: "Sampling options:",
+          hidden: false,
+          type: "boolean",
+        })
+        // probability settings
+        .option("resample-gene-probability", {
+          alias: [],
+          default: 0.01,
+          description: "Probability a gene gets resampled from scratch.",
+          group: "Sampling options:",
+          hidden: false,
+          type: "number",
+        })
+        .option("delta-mutation-probability", {
+          alias: [],
+          default: 0.8,
+          description: "Probability a delta mutation is performed.",
+          group: "Sampling options:",
+          hidden: false,
+          type: "number",
+        })
+        .option("sample-existing-value-probability", {
+          alias: [],
+          default: 0.5,
+          description:
+            "Probability the return value of a function is used as argument for another function.",
+          group: "Sampling options:",
+          hidden: false,
+          type: "number",
+        })
+        .option("crossover-probability", {
+          alias: [],
+          default: 0.8,
+          description:
+            "Probability crossover happens at a certain branch point.",
+          group: "Sampling options:",
+          hidden: false,
+          type: "number",
+        })
+        .option("constant-pool-probability", {
```

Comment:
I am not sure what the current heuristic is for ordering these, but it might make sense to cluster similar methods (e.g., everything related to the constant pool)

Reply:
They are ordered by the comments in between:

random number generator
sampling
mutation
probabilities
gene defaults values

Reply:
For now, it is fine